setup.py

```diff
@@ -43,6 +91,18 @@ def __init__(self, name: str, sourcedir: str = "") -> None:
 
 class CMakeBuild(build_ext):
     def build_extension(self, ext: CMakeExtension) -> None:
+        if not check_cmake():
+            raise ModuleNotFoundError(
+                "No CMake installation found on the system, "
```

Comment:
Suggestion: "No CMake or no CMake >= 3.19 installation found on the system"

Reply:
Changed